src/modules/commander/PreflightCheck.cpp

```diff
@@ -290,15 +290,15 @@ static bool accelerometerCheck(orb_advert_t *mavlink_log_pub, unsigned instance,
 
 			if (accel_magnitude < 4.0f || accel_magnitude > 15.0f /* m/s^2 */) {
 				if (report_fail) {
-					mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL RANGE, hold still on arming");
+					mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL RANGE, hold still on arming.");
 				}
 				/* this is frickin' fatal */
 				success = false;
 				goto out;
 			}
 		} else {
 			if (report_fail) {
-				mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL READ");
+				mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCELEROMETER READ.");
```

Comment:
Capitalization and wording should be updated for consistency with this PR: #10586
Suggested new text: "Preflight Fail: accelerometer read"